.github/workflows/pr-workflow.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Install uv
         uses: astral-sh/setup-uv@v5
       - name: "Generate code"
-        run: make generate
+        run: make generate ALLOW_BREAKING=true
```

Comment:
It feels a bit weird having `ALLOW_BREAKING=true` because we don't really allow them (I know we rely on the buf action). I proposed an alternative that might be less confusing.